handbook/upgrade/upgrade.md

```diff
@@ -219,32 +230,24 @@ As of SystemLink 21.3.2 SystemLink supports using an external PostgreSQL databas
 
 1. Provision a PostgreSQL server or replica set.
 
-1. Install and configure SystemLink 21.3.2.
+1. Install and configure SystemLink 21.5+.
 
 1. Install `nislmigrate` on your new SystemLink Server.
 
 1. Attach the `D:\` volume used to capture data from your original SystemLink server.
 
 1. Run the command `nislmigrate restore --all --secret <your secret> --dir D:\migration`.
 
-1. Open `C:\ProgramData\National Instruments\Skyline\Config\TestMonitor.json`.
-
-1. Change the key `"Postgres.Enabled"` value from `false` to `true`.
-
-1. Add the key `"Postgres.ConnectionString"` and a connection string value. For example:
-
-        "Postgres.ConnectionString" : "Host=<insert db host>;Port=<insert port>;Username=<insert username>;Password=<insert password>;Database=<insert database>;SSL Mode=<Disable or Enable>"
-
-1. Add the key `"Postgres.ConfigurationApproved"=true`.
+1. Open the **NI SystemLink Server Configuration** application.
 
-1. Save and close the file.
+1. Navigate to PostgreSQLDatabase and connect to your external PostgreSQL database.
 
-1. Open the **NI SystemLink Server Configuration** application.
+1. Navigate to TestMonitor and approve the migration.
 
 1. Navigate to **NI SystemLink Service Manager** and click **Restart**.
 
     !!!note
-        After this step SystemLink will migrate your test steps, results, and products from MongoDB to PostgreSQL. Depending on the size of your data set this process may take some time.
+        After this step SystemLink will migrate your test steps, results, and products from MongoDB to PostgreSQL. Depending on the size of your data set this process may take some time. For reference, a typical server takes less than one hour to migrate 5 million steps. To check the step count on your server, you can run the following command in Robo 3T: `db.getCollection('steps').stats()`. Use the step count to roughly estimate the expected migration time. Note that system resources and network connectivity will impact the migration time.
```

Comment:
We are essentially repeating the same note from earlier in the document. Can we consolidate these somehow?

Reply:
To address and your other `!!!note` comment, I opted to create a new section under `Preparing for Upgrade or Migration`. This covers all of the general things users need to know about the migration and eliminates the duplication. Let me know what you think.